canonical/store-components
# Update storybook

## Done
Updated to latest version of storybook

## QA
- Go to https://store-components-85.demos.haus/
- Check that the docs work
